docker/README.md

```diff
@@ -62,12 +62,90 @@ docker-compose -f docker-compose-mysql.yml down
 
 Please note that SQL support is still in active developement and it is not production ready yet.
 
-Quickstart for production
+One host production setup with docker-compose
 =========================
-In a typical production setting, dependencies (cassandra / statsd server) are
-managed / started independently of the cadence-server. To use the container in
-a production setting, use the following command:
+For a simple production setup, start with our **docker-compose-production-example.yml** for a single host Cadence cluster as described in the following steps.
 
+#### 1. Modify the database mounting path in the example YAML file
+You may want to set the Cassandra data location because:
+* a. You need a disk partition that has enough space
+* b. You prefer to have separated storage hardware for have a separated storage hardware for database for performance, backup plan, and availability (with RAID)
+* c. Setting a mounting path can avoid losing data when you run **docker-compose down**
+
+Save the edited example YAML file as docker-compose.yaml.
+
+See the [Docker Documentation](https://docs.docker.com/compose/production/) for more information about using docker-compose in production.
```

Comment:
Seems this section belongs in the section below.

Reply:
Yes, you are right.